projects/aca-content/folder-rules/src/manage-rules/manage-rules.smart-component.spec.ts

```diff
@@ -119,12 +144,12 @@ describe('ManageRulesSmartComponent', () => {
 
     expect(component).toBeTruthy();
 
-    expect(loadRuleSetsSpy).toHaveBeenCalledOnceWith(component.nodeId);
+    expect(folderRuleSetsService.loadRuleSets).toHaveBeenCalledOnceWith(component.nodeId);
 
-    const ruleGroupingSections = debugElement.queryAll(By.css(`[data-automation-id="rule-list-item"]`));
-    const rules = debugElement.queryAll(By.css('.aca-rule-list-item'));
-    const ruleDetails = debugElement.query(By.css('aca-rule-details'));
-    const deleteRuleBtn = debugElement.query(By.css('#delete-rule-btn'));
+    const ruleGroupingSections = unitTestingUtils.getAllByCSS(`[data-automation-id="rule-list-item"]`);
+    const rules = unitTestingUtils.getAllByCSS('.aca-rule-list-item');
```

Comment:
here are couple other places where we use the same selector, please create some wrapping function or move selector to const and reuse